plugins/search-react/src/context/SearchContext.tsx

```diff
@@ -98,9 +100,10 @@ export const useSearchContextCheck = () => {
  */
 const searchInitialState: SearchContextState = {
   term: '',
-  pageCursor: undefined,
-  filters: {},
   types: [],
+  filters: {},
+  pageLimit: undefined,
```

Comment:
Looks like the changeset for this package was lost in that last force push.  ...But in it, I would definitely highlight the fact that this it's possible to set this on the `initialState` prop of the `<SearchContextProvider>` -- I imagine that'll be the primary way that adopters will interact with this new functionality.

Reply:
We are just moving the page limit component to a [separate pull request,](https://github.com/backstage/backstage/pull/13968) the `search-react` changeset is back.